### PR TITLE
made 404 page

### PR DESCRIPTION
**Pull request by almir0201**

### CODE REVIEW

**Comment by almir0201** [comment]
Close #127

File: src/pages/ErrorPage.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            </div>
+            <div className="w-full max-w-[690px] mx-auto">
+              <AspectRatio variant="10/9">
+                <img src={teamRocket} alt="teamRocket" />
+              </AspectRatio>
+            </div>
+          </div>
+        </div>
+        <VerticalLayout justify="center" className="lg:flex-row">
+          {width > 1023 ? (
```
Comment (Shekey) [inline src/pages/ErrorPage.js:42]: div with className hidden md:block will replace this

File: src/pages/ErrorPage.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            <Headline type="h3" className="lg:mr-3">
+              <b>The rocket team</b>
+            </Headline>
+          ) : (
+            <Paragraph variant="2xl" className="text-white" font="font-karla">
+              <b>The rocket team</b>
+            </Paragraph>
+          )}
+          {width > 1023 ? (
```
Comment (Shekey) [inline src/pages/ErrorPage.js:51]: same like above

File: src/pages/ErrorPage.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            </Paragraph>
+          )}
+        </VerticalLayout>
+        <Links
+          o={'..'}
+          onClick={(e) => {
+            navigate(-1);
+          }}>
+          <Button
+            variant={width > 1023 ? 'lg' : 'md'}
```
Comment (Shekey) [inline src/pages/ErrorPage.js:67]: add two buttons with classes hidden md:block and block md:hidden

File: src/pages/ErrorPage.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+              <b>has won this time.</b>
+            </Headline>
+          ) : (
+            <Paragraph variant="2xl" font="font-karla">
+              <b>has won this time.</b>
+            </Paragraph>
+          )}
+        </VerticalLayout>
+        <Links
+          o={'..'}
```
Comment (Shekey) [inline src/pages/ErrorPage.js:62]: what's this O prop?

**Review by Shekey — CHANGES_REQUESTED** [review]

File: src/components/ui/Links/Links.js [hunk 5]
```diff
@@ -1,13 +1,10 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Paragraph } from '../Paragraph/Paragraph';
 
-export const Links = ({ children, to }) => {
+export const Links = ({ children, to, className }) => {
```
Comment (Shekey) [inline src/components/ui/Links/Links.js:4]: also rename this component to singular, it was confusing to me reading this

rename to link

**Review by Shekey — CHANGES_REQUESTED** [review]

File: src/components/header/Header.js [hunk 6]
```diff
@@ -51,11 +52,15 @@ export const Header = () => {
                   ? 'absolute flex flex-col w-full left-0 pt-9 pb-11 top-[100%] gap-4 bg-gradient-to-b from-third to-primary md:relative md:pt-0 md:pb-4 md:flex-row md:gap-8 md:w-auto md:bg-none lg:gap-16'
                   : 'hidden md:flex pb-4 md:gap-8 lg:gap-16'
               }`}>
-              {routes.children.map((route, index) => (
-                <Links key={index} to={route.path}>
-                  {route.title}
-                </Links>
-              ))}
+              {routes.children
+                .filter((route) => route.title !== '404')
```
Comment (Shekey) [inline src/components/header/Header.js:56]: I think now you don't need this, right?

**Review by Shekey — APPROVED** [review]

**Pull request merged**